install/index.adoc

```diff
@@ -51,6 +51,8 @@ configurations.
 xref:multi-masters-using-native-ha[Multiple Masters Using Native HA] example as
 a basis for your environment.
 
+* _Is xref:../install_config/monitoring/cluster_monitoring.adoc[cluster monitoring] required?_ The monitoring stack requires additional xref:../scaling_performance/scaling_cluster_monitoring.adoc[system resources]. Note that by default the monitoring stack is installed. See the xref:../install_config/monitoring/cluster_monitoring.adoc[cluster monitoring documentation] for more information.
```

Comment:
It seems the link is wrong
should be 
/../../install_config/monitoring/cluster_monitoring.adoc
/../../scaling_performance/scaling_cluster_monitoring.adoc
/../../install_config/monitoring/cluster_monitoring.adoc

Reply:
Links will probably change again after the final TOC is confirmed.

Reply:
Finalized TOC, fixed the links